core/server/proxy/src/main/java/alluxio/proxy/s3/signature/SignatureInfo.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.proxy.s3.signature;
+
+/**
+ * Signature and related information.
+ * <p>
+ * Required to create stringToSign and token.
+ */
+public class SignatureInfo {
+  private Version mVersion;
```

Comment:
final

Reply:
Thank you for your time. I will optimize the code with reference to these comments.